app/services/main.py

```diff
@@ -33,8 +35,11 @@
 else:
     app = FastAPI(openapi_url=None, docs_url=None, redoc_url=None)
 
+# mounts an "independent" app on the /docs path that handles static files
+app.mount("/docs", StaticFiles(directory="docs"), name="docs")
 # add the sub-APIs
 app.include_router(an, prefix="/action_network", tags=["action_network"])
+app.include_router(mobilize, prefix="/mobilize", tags=["mobilize"])
```

Comment:
Same silly issue as above: please invert lines 42 and 43 so the includes are alphabetized :).